meshtastic/mesh.proto

```diff
@@ -825,6 +825,11 @@ enum HardwareModel {
    */
   RAK3401 = 117;
 
+  /*
+   * RAKwireless WisBlock Core RAK3112 https://docs.rakwireless.com/product-categories/wisduo/rak3112-module/overview/
+   */
+  RAK3112 = 118;
+
```

Comment:
The RAK 3112 stamp is covered already in it's alter ego RAK3312 Wisblock Module. Confirm ID 106